app/editor/src/features/content/form/ContentForm.tsx

```diff
@@ -1,85 +1,53 @@
-import {
-  Area,
-  Button,
-  ButtonVariant,
-  Col,
-  FormikCheckbox,
-  FormikDropdown,
-  FormikText,
-  IOptionItem,
-  OptionItem,
-  Row,
-  Tab,
-  Tabs,
-} from 'components';
+import { Button, ButtonVariant } from 'components/button';
+import { Col } from 'components/col';
+import { Area, IOptionItem, OptionItem } from 'components/form';
+import { FormikCheckbox, FormikDropdown, FormikText } from 'components/formik';
+import { Row } from 'components/row';
+import { Tab, Tabs } from 'components/tabs';
 import { Formik } from 'formik';
-import { IContentApi, useApiEditor } from 'hooks';
-import React, { useEffect, useState } from 'react';
-import { useNavigate } from 'react-router-dom';
-import { useLookup } from 'store';
+import React from 'react';
+import { useNavigate, useParams } from 'react-router-dom';
+import { useContent, useLookup } from 'store/hooks';
 
 import { PropertiesContentForm } from '.';
+import { defaultFormValues } from './constants';
+import { IContentForm } from './interfaces';
 import { TranscriptContentForm } from './TranscriptContentForm';
-import { formContentToApiContent } from './utils';
+import { toForm, toModel } from './utils';
```

Comment:
This will now be the convention we will use.  Add these utils to the local `./utils` folder to convert back and forth between model and form objects.